add-cjs-extension.cjs

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
```

Comment:
This wouldn't be necessary with the nested `package.json` route. Also, nested directories aren't included here.